src/operator/sequence_reverse-inl.h

```diff
@@ -118,9 +115,25 @@ class SequenceReverseOp : public Operator {
     const index_t other_dim = data.size(2);
     const index_t tensor_numel = data.shape_.Size();
 
-    mxnet_op::Kernel<ReverseKernel, xpu>::Launch(
-        s, max_seq_len, out.dptr_, data.dptr_, req, max_seq_len, batch_size,
-        other_dim, tensor_numel, indices);
+    switch (req) {
```

Comment:
Oh there is `MXNET_ASSIGN_REQ_SWITCH`.. but this is fine, too

Reply:
I did not notice that, will change and rebase.